### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Yemisirach**
- [ ] Add a contact section in an index.html and style well by fulfilling the design requirement.
-Name field accepts up to 30 chars.
-Email field accepts only emails.
-Message field accepts up to 500 chars.
-All fields have placeholder text.
-All fields have the correct type.
-A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -79,17 +79,15 @@
...
   height: 160px;
   font-size: 40px;
   font-weight: 700;
-  padding-top: 90px;
-  padding-left: 24px;
-  padding-right: 24px;
+  padding: 90px 24px 0 24px;
   font-family:
     "Lucida Sans",
     "Lucida Sans Regular",
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Great job 💪. Kindly make sure you use the correct `font-family` in your site as seen in your chosen Figma design 🙏

File: styles.css [hunk 2]
```diff
@@ -330,6 +331,19 @@ ul {
   width: 327px;
 }
 
+.Get_btn button {
+  margin-top: 222px;
+  height: 50px;
+  width: 108px;
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Kindly make sure the `button` in your Contact Me section looks as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Yemisirach , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️

- [ ] Great job with your `README` file 👍. Kindly make sure your `README` file has all the headers and sections as seen in the [template](https://github.com/microverseinc/readme-template) provided by Microverse 🙏 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yemisirach** [comment]
-Adjust the get in touch button based on the review. And the 
Poppins font family is not included as a font family. why because I use that in front of Poppins plus the linter has executed an error on the unavailable font family.

**Review by emmiiorji — APPROVED** [review]
Hi @LethaboMot and @Ezzywealth ,

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yemisirach** [comment]
- I Use media queries in your CSS file to make the desktop version look like the template you selected (Desktop: 768px or wider).
- In order to lay out the elements on the page I should use Grid and Flexbox.

- [ ]   I  would like to tell you about something an issue on a branch. The contacts branch has been approved but I do the desktop version on the same branch so I diced to merge the code after when the desktop version review is approved it is ok?

**Review by emmiiorji — COMMENTED** [review]
Hi @Yemisirach, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted an already approved pull request for a different milestone. Kindly note that a new pull request containing the changes is required for each milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**